ocs_ci/deployment/baremetal.py

```diff
@@ -1543,6 +1543,222 @@ def clean_disk(
         logger.info(out)
 
 
+def detect_simulation_disk_on_node(wnode, namespace="default", timeout=300):
+    """
+    Detects the last available /dev/sd* disk on a given worker node.
+
+    Args:
+        wnode (ocs_ci.ocs.resources.ocs.OCS): The worker node object.
+        namespace (str): Namespace for the debug pod.
+        timeout (int): Timeout for the command execution.
+
+    Returns:
+        str or None: The detected disk path (e.g., "/dev/sdb") or None if not found.
+
+    """
+    logger.info("Attempting to auto-detect a suitable /dev/sd* disk.")
```

Comment:
```suggestion
    logger.info(f"Attempting to auto-detect a suitable /dev/sd* disk on worker node: {wnode.name}.")
```
Is it worth to mention a worker node here?